spec/unit/hanami/cli/commands/app/generate/struct_spec.rb

```diff
@@ -0,0 +1,117 @@
+# frozen_string_literal: true
+
+RSpec.describe Hanami::CLI::Commands::App::Generate::Struct, :app do
+  subject { described_class.new(fs: fs, inflector: inflector, out: out) }
+
+  let(:out) { StringIO.new }
+  let(:fs) { Hanami::CLI::Files.new(memory: true, out: out) }
+  let(:inflector) { Dry::Inflector.new }
+  let(:app) { Hanami.app.namespace }
+  let(:dir) { inflector.underscore(app) }
+
+  def output
+    out.rewind && out.read.chomp
+  end
```

Comment:
```suggestion
  def output
    out.string.chomp
  end
```

I realised there's a way nicer way to do this that avoids mutating the IO object itself.

Reply:
Sweet. Did this for struct and operation, and made a note to self to fix these for the other generators when I switch them over to using this RubyFileGenerator/Writer